### ISSUE TITLE
SignOut/Disconnect exception:

### ISSUE THREAD

**Issue by chmiiller** [AppWerft/Ti.GoogleSignIn#3]
When trying to signOut in the same activity I'm receiving this error:
```
java.lang.IllegalStateException: GoogleApiClient is not connected yet.
    at com.google.android.gms.internal.zzaat.zzb(Unknown Source)
	at com.google.android.gms.auth.api.signin.internal.zze.zza(Unknown Source)
	at com.google.android.gms.auth.api.signin.internal.zzc.signOut(Unknown Source)
	at ti.googlesignin.GooglesigninModule.signOut(GooglesigninModule.java:138)
	at org.appcelerator.kroll.runtime.v8.V8Object.nativeFireEvent(Native Method)
	at org.appcelerator.kroll.runtime.v8.V8Object.fireEvent(V8Object.java:62)
	at org.appcelerator.kroll.KrollProxy.doFireEvent(KrollProxy.java:872)
	at org.appcelerator.kroll.KrollProxy.handleMessage(KrollProxy.java:1095)
	at org.appcelerator.titanium.proxy.TiViewProxy.handleMessage(TiViewProxy.java:354)
	at android.os.Handler.dispatchMessage(Handler.java:98)
	at android.os.Looper.loop(Looper.java:148)
	at org.appcelerator.kroll.KrollRuntime$KrollRuntimeThread.run(KrollRuntime.java:118)
```

[With what I've found on Stack Overflow](http://stackoverflow.com/questions/29343922/googleapiclient-is-throwing-googleapiclient-is-not-connected-yet-after-onconne) seems to be a problem with `GoogleApiClient googleApiClient` not found on the signOut method. Do you have any idea on how to solve it? I've tried to make it static, but it also doesn't seem to work.

Thanks!

**Comment by AppWerft** [comment]
Hi, in new version 1.0.7 the signout process asks by client and escapes in case of null

**Comment by chmiiller** [comment]
Gonna test it and let you know! Thanks!

**Comment by chmiiller** [comment]
Not fixed yet 😞
Still the same. Tested with 1.0.7 and 1.1.2

**Comment by chmiiller** [comment]
Fixed on 1.1.3: https://github.com/AppWerft/Ti.GoogleSignIn/releases/tag/1.1.3
